include/function_dependencies.php

```diff
@@ -46,20 +46,26 @@ function check($binary)
     $retval = 0;
     $logger->info(__FUNCTION__.' : checking '.$binary);
     
-	$retval = execCMD('exiftool -ver'); // redirect any output
```

Comment:
these checks do not redirect any output.  The output were being sent to the browser